package.json

```diff
@@ -15,7 +16,7 @@
     "private": true,
     "type": "module",
     "scripts": {
-        "sync": "node sync-addresses.js > ./resources/addresses.md",
+        "sync": "node sync-addresses.js > ./resources/addresses.md && node sync-partials.js",
```

Comment:
Should we run this in a commit hook?

Reply:
Or should we run this in CI and assert there is no diff?

Or do that in a cron job?